db/009-rename-waiting-for-response.sql

```diff
@@ -0,0 +1,23 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 9;
```

Comment:
@sampathsatti , our PRs conflict right now on this number. We'll just need to remember to change it when we figure out merge order.